packages/components/organisms/f-offers/stories/Offers.stories.js

```diff
@@ -103,7 +103,8 @@ export const VOffersComponent = (args, { argTypes }) => ({
 VOffersComponent.storyName = 'f-offers';
 
 VOffersComponent.args = {
-    brazeApiKey: '__TEST_API_KEY__'
+    authToken: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6ImxiRmxsR2dSVTRUaldlTEJxVzlacDNBNW00USIsImtpZCI6ImxiRmxsR2dSVTRUaldlTEJxVzlacDNBNW00USJ9.eyJpc3MiOiJodHRwczovL2FsbC1hdXRob3JpemVhcGktc3RhZ2luZy11ay5zdGFnaW5nLXVrLmplLWxhYnMuY29tL2lkZW50aXR5IiwiYXVkIjoiaHR0cHM6Ly9hbGwtYXV0aG9yaXplYXBpLXN0YWdpbmctdWsuc3RhZ2luZy11ay5qZS1sYWJzLmNvbS9pZGVudGl0eS9yZXNvdXJjZXMiLCJleHAiOjE2MzA0ODgwMzksIm5iZiI6MTYzMDQ4NDQzOSwiY2xpZW50X2lkIjoid2ViX3NvY2lhbCIsInJlYWxtX2lkIjoiYTNjMjQ2ZTEtMjRhNC00NWE2LWEzNjItZmRhNzNhNGI3NWFiIiwic2NvcGUiOlsib3BlbmlkIiwib2ZmbGluZV9hY2Nlc3MiLCJtb2JpbGVfc2NvcGUiXSwic3ViIjoiMTc5ODQ3MDgiLCJhdXRoX3RpbWUiOjE2Mjg1MjA1MTAsImlkcCI6Imlkc3J2IiwiZW1haWwiOiJtYXR0aGV3aGFyZGVybkBnbWFpbC5jb20iLCJjcmVhdGVkX2RhdGUiOiIyMDIwLTEwLTE1VDA5OjI5OjE2LjAzMDAwMDBaIiwibmFtZSI6Im1hdHRoZXcgaGFyZGVybiIsImdsb2JhbF91c2VyX2lkIjoiVTB0WlV4QWhvKzZkSGs3UC9kcmRlUlJ2emVvPSIsImdpdmVuX25hbWUiOiJtYXR0aGV3Iiwicm9sZSI6IlJlZ2lzdGVyZWQiLCJmYW1pbHlfbmFtZSI6ImhhcmRlcm4iLCJqZXQiOiJINGVpcVZBTjJrMmgwRzIxUUQwZ3ZRIiwiZ3JhbnRfdHlwZSI6InJlZnJlc2hfdG9rZW4iLCJhbXIiOlsiZGVsZWdhdGlvbiJdfQ.RcnhqcxOsY-FKwdyd4mPoGi3gZi20Gq1mGYamku3qLFeg_EyMXO6Rijd8lW2w0d42AXTnyaPznllyk97fnSglIWshUnlEmIsKsB24KC1sJ6TOQFxVdcahMA9UNqVJNFT87Rjp-NNLOwNF31YtE6oR10im_4aWc2pprjdm7nVZJWJR8XZC2bxjnQfjY0FbTPs__wbjbX73kU3LltIPgzAcWoyjPtx01SSqBGHnuG0VDrsP-Lhl3vZtETS89CZ5IK7yFWP1uRaPgG1R2SEUytRM73uHVKJ2qldnDnvi31TxYwWNnzR1OIWwpl2UgnWg-0peNXF0c4qbiRLfxhWY9bYHb4_i547TNQ7xKEJfleJmus0YpS1z2Rd1UDsapHPKqCAFF5cZMV_MjQ8Yf3lP9l-sO6l4w3_uH2RPw28pGcRdxfws5ecvra5KsoUYXl4usOw_tZ0YykRk7Fk6vVVoIuR8edbKU_uODzL55vx3L8dt86v1EYzgLdwSVF5l_n8R35K_aQkin4XUcx-OO_Z5NF_LQsqz_4FBlEWrGHoQP2b8FCOfHj6MP4OFo0-ycS_x5_r0rfQ03j8ZyCT04399NyUnzy26EC6JLaoqCu6NhZIacrk_OPvn2k-zqqyIQ4ShAcs5JIjLezx7Ot658bqrHqymglEXfL1Bsktk6wPDHCg8Ho',
```

Comment:
Are we OK to add these values in a public repo?

Reply:
no concerns per say as its not valid, but I will remove as it should be `__TEST_TOKEN__` anyway so thanks for the catch.